### PR TITLE
feat(vara): Store dust in treasury

### PR DESCRIPTION
**Pull request by ukint-vs**
Dust will move into treasury pot.

@reviewer-or-team

### CODE REVIEW

**Review by breathx — CHANGES_REQUESTED** [review]
Any test?

**Review by ekovalev — APPROVED** [review]

File: runtime/vara/src/integration_tests.rs [hunk 1]
```diff
@@ -61,11 +61,13 @@ pub(crate) fn on_initialize(new_block_number: BlockNumberFor<Runtime>) {
     GearMessenger::on_initialize(new_block_number);
     Gear::on_initialize(new_block_number);
     GearGas::on_initialize(new_block_number);
+    Treasury::on_initialize(new_block_number);
     // Session::on_initialize(new_block_number);
 }
 
 // Run on_finalize hooks (in pallets reverse order, as they appear in AllPalletsWithSystem)
 pub(crate) fn on_finalize(current_blk: BlockNumberFor<Runtime>) {
+    Treasury::on_initialize(current_blk);
```
Comment (breathx) [inline runtime/vara/src/integration_tests.rs:70]:
```suggestion
    Treasury::on_finalize(current_blk);
```

Comment (breathx) [reply runtime/vara/src/integration_tests.rs:70]: and keep pallets ordering pls

**Review by breathx — APPROVED** [review]

**Review by breathx — CHANGES_REQUESTED** [review]
Please move dust into offset pool

**Pull request closed without merging**